src/scripts/quests/QuestLineHelper.ts

```diff
@@ -766,6 +766,65 @@ class QuestLineHelper {
         App.game.quests.questLines().push(galacticSinnohQuestLine);
     }
 
+    //Giratina quest
+    public static createGiratinaQuestLine() {
+        const giratinaQuestLine = new QuestLine('Zero\'s Ambition', 'Help Zero find an entrance to the Distortion World.', new MultiRequirement([new ObtainedPokemonRequirement(pokemonMap.Uxie), new ObtainedPokemonRequirement(pokemonMap.Mesprit), new ObtainedPokemonRequirement(pokemonMap.Azelf), new GymBadgeRequirement(BadgeEnums.Elite_SinnohChampion)]), GameConstants.BulletinBoards.Sinnoh);
+
+        const talktoZero1 = new TalkToNPCQuest(FightAreaZero1, 'Talk to Zero in the Fight Area.');
+        giratinaQuestLine.addQuest(talktoZero1);
+
+        const readSinnohMyth = new TalkToNPCQuest(CanalaveSinnohMyth, 'Read the recently discovered book in Canalave City.');
+        giratinaQuestLine.addQuest(readSinnohMyth);
+
+        const talktoZero2 = new TalkToNPCQuest(FightAreaZero2, 'Report to Zero what was in the book.');
+        giratinaQuestLine.addQuest(talktoZero2);
+
+        const talktoMesprit = new TalkToNPCQuest(VerityMesprit, 'Ask Mesprit about the Distorsion World in Lake Verity.');
+        giratinaQuestLine.addQuest(talktoMesprit);
+
+        const obtain10PurpleShards = new CustomQuest(10, undefined, 'Obtain 10 Purple Shards.', () => player.mineInventory().find(item => item.name == 'Purple Shard').amount());
+        giratinaQuestLine.addQuest(obtain10PurpleShards);
+
+        const talktoAzelf = new TalkToNPCQuest(ValorAzelf, 'Ask Azelf about the Distorsion World in Lake Valor.');
+        giratinaQuestLine.addQuest(talktoAzelf);
+
+        const obtain10OchreShards = new CustomQuest(10, undefined, 'Obtain 10 Ochre Shards.', () => player.mineInventory().find(item => item.name == 'Ochre Shard').amount());
+        giratinaQuestLine.addQuest(obtain10OchreShards);
+
+        const talktoUxie = new TalkToNPCQuest(AcuityUxie, 'Ask Uxie about the Distorsion World in Lake Acuity.');
```

Comment:
Distorsion -> Distortion
```suggestion
        const talktoMesprit = new TalkToNPCQuest(VerityMesprit, 'Ask Mesprit about the Distortion World in Lake Verity.');
        giratinaQuestLine.addQuest(talktoMesprit);

        const obtain10PurpleShards = new CustomQuest(10, undefined, 'Obtain 10 Purple Shards.', () => player.mineInventory().find(item => item.name == 'Purple Shard').amount());
        giratinaQuestLine.addQuest(obtain10PurpleShards);

        const talktoAzelf = new TalkToNPCQuest(ValorAzelf, 'Ask Azelf about the Distortion World in Lake Valor.');
        giratinaQuestLine.addQuest(talktoAzelf);

        const obtain10OchreShards = new CustomQuest(10, undefined, 'Obtain 10 Ochre Shards.', () => player.mineInventory().find(item => item.name == 'Ochre Shard').amount());
        giratinaQuestLine.addQuest(obtain10OchreShards);

        const talktoUxie = new TalkToNPCQuest(AcuityUxie, 'Ask Uxie about the Distortion World in Lake Acuity.');
```